spinetoolbox/spine_db_editor/widgets/spine_db_editor.py

```diff
@@ -416,6 +419,11 @@ def update_commit_enabled(self, _clean=False):
         self.ui.actionRollback.setEnabled(dirty)
         self.setWindowModified(dirty)
         self.windowTitleChanged.emit(self.windowTitle())
+        self.url_toolbar.reload_action.setEnabled(not dirty)
+        if dirty:
+            self.url_toolbar.reload_action.setIcon(self._disabled_reload_icon)
+        else:
+            self.url_toolbar.reload_action.setIcon(self._enabled_reload_icon)
```

Comment:
Why do we need to set the gray icon manually? Isn't disabling the action enough?

Reply:
At least on my machine somehow that doesn't work for the Reload button. The Commit and Rollback buttons do however turn grey just by disabling the actions